Valigator.AspNetCore.Newtonsoft.Json/MvcBuilderExtensions.cs

```diff
@@ -53,10 +57,15 @@ public static IMvcBuilder AddValigator(this IMvcBuilder builder, Func<AspNetCore
 					options.Filters.Add(new ValigatorResultFilter(resultErrorCreator));
 				})
 #if NETCOREAPP3_0
-				.AddNewtonsoftJson(o => o.SerializerSettings.Converters.Add(new ValigatorConverter()));
+				.AddNewtonsoftJson(o => o.SerializerSettings.Converters.Add(new ValigatorConverter(o.SerializerSettings)));
```

Comment:
We need to pass in the SerializerSettings because we create a new serializer in the converter. This is due to an annoying "feature" of Newtonsoft.Json where it records deserialization errors and throws them later in the process where we cannot catch them.